packages/cmf-cqrs/src/middleware/smartWebsocket.js

```diff
@@ -1,3 +1,5 @@
+import { setTimeout } from 'timers';
```

Comment:
Why do you need this ?
I don't see any package.json modification, is it already availalble as a dependency ?